src/main/java/com/kruize/environment/docker/DockerEnvImpl.java

```diff
@@ -29,24 +40,84 @@ public void setupMonitoringAgent()
     @Override
     public void setupApplicationRecommendations()
     {
-
+        this.applicationRecommendations = DockerApplicationRecommendations.getInstance();
     }
 
     @Override
     public void setupAnalysis()
     {
+        this.analysis = DockerAnalysisImpl.getInstance();
+    }
 
+    @Override
+    public void setupQuery()
+    {
+        this.query = PrometheusQuery.getInstance();
     }
 
     @Override
     public void getAllApps()
     {
+        JsonArray containerList = null;
+        try (FileReader reader = new FileReader("/opt/app/kruize-docker.json"))
```

Comment:
Can you add some documentation about this file and the format that is expected. Example file can be pasted here for reference.